### ISSUE TITLE
Documentation Request: Filled/Outlined Text Fields (new Material Design)

### ISSUE THREAD

**Issue by lacolaco** [angular/material2#11429]
#### Bug, feature request, or proposal:

Documentation Request

#### What is the expected behavior?

https://material.angular.io/components/form-field/overview

Some documentation and examples for Filled/Outlined text field.

#### What is the current behavior?

No documentation

**Comment by rafaelss95** [comment]
Aren't you looking for `appearance` input in `<mat-form-field>`? Check It https://material.angular.io/components/form-field/api#MatFormField

**Comment by lacolaco** [comment]
No. It's outdated design. Please check the latest Material Design guide. 
https://material.io/design/components/text-fields.html

**Comment by lacolaco** [comment]
https://github.com/angular/material2/blob/c9bd05c5b9084ad5e6b23670220fac7b64c5a9ee/src/lib/form-field/form-field.ts#L69

It's not documented so I didn't know it. Thank you for the information.

**Comment by rafaelss95** [comment]
You can leave it open requesting some docs then.

**Comment by LoganDark** [comment]
![image](https://user-images.githubusercontent.com/4723091/40313357-69bfd90a-5cca-11e8-9654-967620349f52.png)
Go home GitHub, you're drunk. :P
